tests/models/visual_bert/test_modeling_visual_bert.py

```diff
@@ -196,15 +196,18 @@ def prepare_config_and_inputs_for_multiple_choice(self):
             labels = ids_tensor([self.batch_size], self.num_choices)
 
         config = self.get_config()
-        return config, {
-            "input_ids": input_ids,
-            "token_type_ids": token_type_ids,
-            "attention_mask": attention_mask,
-            "visual_embeds": visual_embeds,
-            "visual_token_type_ids": visual_token_type_ids,
-            "visual_attention_mask": visual_attention_mask,
-            "labels": labels,
-        }
+        return (
+            config,
+            {
+                "input_ids": input_ids,
+                "token_type_ids": token_type_ids,
+                "attention_mask": attention_mask,
+                "visual_embeds": visual_embeds,
+                "visual_token_type_ids": visual_token_type_ids,
+                "visual_attention_mask": visual_attention_mask,
+                "labels": labels,
+            },
+        )
```

Comment:
Not sure why this change is here? It's not related to this PR.

Reply:
As mentioned previously, my local `black` seems to have conflicting configs and this was formatted automatically.

That said, the [code-quality test isn't passing](https://app.circleci.com/pipelines/github/huggingface/transformers/41068/workflows/d195a0da-e6e0-4b24-8d28-ea37093f05e4/jobs/467262?invite=true#step-106-5) at this file, which did pass before I reverted this change.

Not sure why even circleci's `black` is also complaining about this?

Reply:
and then run `make style` again?